tools/please_go_install/main.go

```diff
@@ -108,114 +104,70 @@ func (g *pkgGraph) compile(from []string, target string) {
 	from = checkCycle(from, target)
 
 	pkgDir := filepath.Join(opts.SrcRoot, target)
-
 	// The package name can differ from the directory it lives in, in which case the parent directory is the one we want
 	if _, err := os.Lstat(pkgDir); os.IsNotExist(err) {
 		pkgDir = filepath.Dir(pkgDir)
 	}
 
-	packageASTs, err := parser.ParseDir(fileSet, pkgDir, nil, 0)
+	// TODO(jpoole): is import vendor the correct thing to do here?
+	pkg, err := build.ImportDir(pkgDir, build.ImportComment)
```

Comment:
You could just pass `0` here. Do you need the comments for something (I believe the relevant cgo comments are parsed anyway, but I could be wrong)

Reply:
Ah, I was wondering if this was responsible for parsing the linker flags and such. Will have to investigate.